packages/terra-table/src/proptypes/columnShape.js

```diff
@@ -42,6 +42,18 @@ const columnShape = PropTypes.shape({
    * One of `ascending`, `descending`.
    */
   sortIndicator: PropTypes.oneOf(Object.values(SortIndicators)),
+
+  /**
+   * @private
+   * The color to be used for highlighting a column.
+   */
+  columnHighlightColor: PropTypes.oneOf(['orange', 'green']),
+
+  /**
+   * @private
+   * The information to be conveyed to screen readers about the highlighted column.
+   */
+  columnHighlightDescription: PropTypes.string,
```

Comment:
Shouldn't these be at the Table and FlowsheetDataGrid levels?

Reply:
No, this would be exposed only through the column object under flowsheet. This is to support multi-column highlight.